visallo/visallo
# remove rdf import capability

Testing Instructions:
- Make docs don't reference rdf 
- Make sure [getting started](http://docs.visallo.org/getting-started.html) still work

Points of Regression: N/A

CHANGELOG
Removed: Rdf Import plugin
